bentoml/yatai/client/bento_repository_api.py

```diff
@@ -344,62 +346,62 @@ def list(
             raise BentoMLException(f'{error_code}:{error_message}')
         return result.bentos
 
-    def delete(self, bento):
+    def delete(
+        self,
+        all=False,  # pylint: disable=redefined-builtin
+        confirm_delete=False,
```

Comment:
It's good to make it `False` here by default in the Python API, but we may want to make it `True` by default in the CLI